### DISCUSSION TITLE
如果不从Crypt里下载文件，直接从网盘里下载加密过的文件，自己用什么方法才能解密打开？

### DISCUSSION THREAD

**Discussion by sanliuyi901** [AlistGo/alist#5620] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [ ] I'm sure there are no duplicate issues or discussions.
- [ ] I'm sure this feature is not implemented.
- [ ] I'm sure it's a reasonable and popular requirement.

### Description of the feature / 需求描述

如果不从Crypt里下载文件，直接从网盘里下载加密过的文件，自己用什么方法才能解密打开？

### Suggested solution / 实现思路

如果不从Crypt里下载文件，直接从网盘里下载加密过的文件，自己用什么方法才能解密打开？

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by theRainight** [comment]
本质使用了rclone的加密方式，只要使用rclone创建一个相同加密方式的目录，那把文件放进去就能解密文件了

**Reply by sanliuyi901** [reply]
如果不用rclone可以解密文件吗

**Comment by anwen-anyi** [comment]
是这样的，上面大佬说的对，如果你对 Rclone 很了解可以看 👉 [Rclone Crypt 文档](https://rclone.org/crypt) 目前完全兼容

**Comment by sd14** [comment]
本地安装AList，添加本机存储，再添加相同加密方式的crypt

**Comment by lhuanyun** [comment]
如果有人写了一个专门用于解密程序，记得踢我一下，

**Reply by pongfcnkl** [reply]
不用写吧 ，上面说的rclone 还有把网盘路径换成本机存储路径都可以

**Reply by Durianyang** [reply]
> 如果有人写了一个专门用于解密程序，记得踢我一下，

踢，https://github.com/huhuzilla/AList-Crypt-Decryption
